pkg/plugin/plugin.go

```diff
@@ -100,6 +109,42 @@ func (p *Plugin) SetInformation(parsedURL *url.URL) error {
 	return nil
 }
 
+func findPluginByName(name string, plugins []store.Plugin) *store.Plugin {
+	// for each plugin in the plugins list, check if the name matches the name of the plugin
+	for _, plugin := range plugins {
+		if plugin.Name == name {
```

Comment:
can you modify this, to check the name AND the author ??
It will allow us more flexibilité (by instance having 2 plugins with same name but differents author)